### PR TITLE
72 signup login

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to
#72 

I left the last point in issue #71 as I'm not able to use add the logout function on the logout nva item,
Hey @fadeomar 
if you can do it later please, the backend is ready to use
=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. 

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/pages/Signup/index.js [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+              <T.P weight={'semi'}>
+                I agree to the{' '}
+                <T.Link
+                  external
+                  underline
+                  color="neutralMain"
+                  weight="semi"
+                  to={R.EXTERNAL.TERMS_OF_USE}
+                >
+                  Sign up
```
Comment (fadeomar) [inline client/src/pages/Signup/index.js:292]:
```suggestion
                 Terms of use
```

**Review by fadeomar — CHANGES_REQUESTED** [review]
update this, so it should direct the user to signup page 
```
        <Col w={[4, 11, 6]}>
          <TextWithIcon
            to={R.ADMIN.WELCOME1}
            text="Don’t have an account? Sign up"
            icon="forwardArrow"
            iconColor="primaryMain"
          />
        </Col>
```

File: client/src/pages/Signup/index.js [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+          />
+        </Col>
+      </Row>
+      <Row mt="7">
+        <Col w={[4, 11, 6]}>
+          <I.Checkbox
+            checked={agreedOnTerms}
+            handleChange={(v) => setState({ agreedOnTerms: v })}
+            label={
+              <T.P weight={'semi'}>
```
[the remark this answers is not in the document]
Comment (RamyAlshurafa) [reply client/src/pages/Signup/index.js:283]: done

File: client/src/pages/Signup/index.js [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+      if (error.name === 'ValidationError') {
+        setState({ validationErrs: error.inner });
+      }
+      return false;
+    }
+  };
+
+  const handleSignup = async () => {
+    setState({ loading: true });
+    const { error, data } = await Users.signup({
```
[the remark this answers is not in the document]
Comment (RamyAlshurafa) [reply client/src/pages/Signup/index.js:124]: done

File: client/src/pages/Signup/index.js [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+    setState({ loading: false });
+
+    if (error) {
+      if (error.statusCode === 409) {
+        setState({ validationErrs: { [error.data.field]: error.message } });
+      } else {
+        setState({ httpError: error.message });
+      }
+    } else {
+      history.push(R.ADMIN.FILL_ORGANISATION_DETAILS);
```
[the remark this answers is not in the document]
Comment (RamyAlshurafa) [reply client/src/pages/Signup/index.js:144]: done

**Comment by RamyAlshurafa** [comment]
> update this, so it should direct the user to signup page
> 
> ```
>         <Col w={[4, 11, 6]}>
>           <TextWithIcon
>             to={R.ADMIN.WELCOME1}
>             text="Don’t have an account? Sign up"
>             icon="forwardArrow"
>             iconColor="primaryMain"
>           />
>         </Col>
> ```

thanks @fadeomar but I think this is meant to be link this, its like that in figma

**Pull request merged**